I3uckwheat/cards-of-carousal
# Add carousel feature and revised copy to pregame welcome text

#### 0. Make sure your issue is linked:

Closes #177 

#### 1. Purpose:

Rewrite the pregame screen welcome/rules text.


#### 2. Implementation:

Because of space limitations I implemented a basic carousel that shows some welcome text at first and an optional full explainer of the rules afterward. 

Because this was making `HostPregameScreen` pretty convoluted I broke this off into its own component.


#### 3. Possible Issues:

The carousel is currently set to flip every two seconds, this is just for carousel demo purposes and should be changed immediately before or after merging this PR.

#### 4. How to Test:


0. Fetch my branch and switch to it
1. Start the application backend then frontend
2. Open one browser window and go to the app
3. Select host mode
4. Verify that what was the lorem ipsum text previously switches between the carousel slides ~2 seconds
5. Click one of the indicator cards and verify that it switches to that specific slide
6. Verify that that the text no longer toggles every two seconds


#### 5. Cleanup
